internal/frameworkprovider/service_resource_model.go

```diff
@@ -12,17 +12,68 @@ import (
 
 // ServiceResourceModel describes the [ServiceResource] data model.
 type ServiceResourceModel struct {
-	Name        string            `tfsdk:"name"`
-	DisplayName types.String      `tfsdk:"display_name"`
-	Project     string            `tfsdk:"project"`
-	Description types.String      `tfsdk:"description"`
-	Annotations map[string]string `tfsdk:"annotations"`
-	Labels      Labels            `tfsdk:"label"`
-	Status      types.Object      `tfsdk:"status"`
+	Name             string                `tfsdk:"name"`
+	DisplayName      types.String          `tfsdk:"display_name"`
+	Project          string                `tfsdk:"project"`
+	Description      types.String          `tfsdk:"description"`
+	Annotations      map[string]string     `tfsdk:"annotations"`
+	Labels           Labels                `tfsdk:"label"`
+	ResponsibleUsers ResponsibleUsersModel `tfsdk:"responsible_user"`
+	ReviewCycle      *ReviewCycleModel     `tfsdk:"review_cycle"`
+	Status           types.Object          `tfsdk:"status"`
+}
+
+// ResponsibleUserModel represents [v1alphaService.ResponsibleUser].
+type ResponsibleUserModel struct {
+	Id types.String `tfsdk:"id"`
+}
+
+type ResponsibleUsersModel []ResponsibleUserModel
```

Comment:
do we need this extra model? I don't think so.
It would be a bit cleaner imo to have the `ResponsibleUserModel` define `ToManifest` method etc.